apikey-service/src/main/java/eu/europeana/apikey/controller/ApikeyController.java

```diff
@@ -386,7 +358,22 @@ public ApikeyController(ApikeyRepo apikeyRepo) {
         this.apikeyRepo.save(apikey);
```

Comment:
I assume that this method might throw a runtime exception that is not handled....